plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java

```diff
@@ -268,6 +283,32 @@ protected IStatus validate() {
 		}
 	}
 
+	private void createOpenShiftConnectionControl(Composite parent, DataBindingContext dbc) {
+		Label lblConnection = new Label(parent, SWT.NONE);
+		lblConnection.setText("OpenShift Connection: ");
+		GridDataFactory.fillDefaults()
+			.align(SWT.FILL, SWT.CENTER)
+			.applyTo(lblConnection);
+		final Text connectionText = new Text(parent, SWT.BORDER | SWT.READ_ONLY);
+		GridDataFactory.fillDefaults()
+			.align(SWT.FILL, SWT.CENTER)
+			.span(2, 1)
+			.grab(true, false)
+			.applyTo(connectionText);
+		final IObservableValue connnectionTextObservable = WidgetProperties.text(SWT.None).observe(connectionText);
+		final IObservableValue connnectionObservable = BeanProperties.value(IDeployImagePageModel.PROPERTY_CONNECTION).observe(model);
+		ValueBindingBuilder.bind(connnectionTextObservable)
+			.to(connnectionObservable)
```

Comment:
I'd add .notConvertingParticipant() here so that no update strategy text -> model is being attached.